src/components/PageHeader.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+    <h1>Welcome to todo app!</h1>
+</template>
+
+<script>
+  import { Vue, Component} from 'vue-property-decorator';
+
+  @Component
+  export default class PageHeader extends Vue {
+    
+  }
+</script>
+
+<style>
+    h1 {
+        font-size: 40px;
+        font-family: 'Times New Roman', Times, serif;
```

Comment:
A global css definition could be useful, font-family and such can be defined there.